app/usecases/beatmap.py

```diff
@@ -172,7 +182,11 @@ async def md5_from_api(md5: str, should_save: bool = True) -> Optional[Beatmap]:
             return beatmap
 
 
-async def id_from_api(id: int, should_save: bool = True) -> Optional[Beatmap]:
+async def id_from_api(
+    id: int,
+    *,
+    should_replace_db_beatmap: bool = False,
```

Comment:
`True` is the dangerous case, so want to make it _more difficult_ to enact/make it so the developer needs to acknowledge it more